packages/address-mocks/src/index.ts

```diff
@@ -1,48 +1,59 @@
-import {GRAPHQL_ENDPOINT} from '@shopify/address-consts';
+import {
+  GRAPHQL_ENDPOINT,
+  LoadCountriesResponse,
+  LoadCountryResponse,
+  ResponseError,
+} from '@shopify/address-consts';
 import {fetch} from '@shopify/jest-dom-mocks';
 
 import {fixtures} from './fixtures';
 
 export * from './fixtures';
 
-interface Options {
-  method?: string;
-  body?: string;
-}
+const OPERATION_NAMES = Object.keys(fixtures);
+const LOCALES = Object.keys(fixtures.countries);
 
 export function mockCountryRequests() {
-  const locales = [
-    'AF',
-    'DA',
-    'DE',
-    'EN',
-    'ES',
-    'FR',
-    'IT',
-    'JA',
-    'NL',
-    'PT',
-    'PT_BR',
-  ];
-  locales.concat(locales).map((locale) => {
-    ['countries', 'country'].map((operationName) => {
-      fetch.mock(
-        (url: string, options: Options) => {
-          if (url !== GRAPHQL_ENDPOINT || options.method !== 'POST') {
-            return false;
-          }
-          if (typeof options.body === 'string') {
-            const body = JSON.parse(options.body);
-            return (
-              body.operationName === operationName &&
-              body.variables.locale === locale
-            );
-          }
-          return false;
-        },
-        fixtures[operationName][locale],
-        {overwriteRoutes: false},
+  fetch.post(
```

Comment:
It's a bug fix.

It doesn't change how consumers are using it, though if they were calling `getCountry('JP')`, they would actually get Canada instead of Japan, unless the whole country list was in the cache.

Reply:
Though it's definitely a minor change (non-breaking but adds new features like error messages)

Reply:
Thanks for the clarification, a change that would surface incorrect usage is minor/patch to me :)